misc/git_add_patch.talon

```diff
@@ -0,0 +1,20 @@
+app: /.*terminal/
+app: /.*term/
+app: Cmd.exe
+title: /git add .*\-p/
+-
+yank:
```

Comment:
Will re-using the alphabet here cause any issues if people need to actually spell something? I don't really use the commandline git.

Reply:
The only options during a `git add -p` are single letters or question mark.